test/runtests.jl

```diff
@@ -1,9 +1,9 @@
-using ImageInTerminal, Images, TestImages, Base.Test, ColorTypes, FixedPointNumbers
+using Base.Test, ImageInTerminal, Images, TestImages,  ColorTypes, FixedPointNumbers
 
 # check for ambiguities
 refambs = detect_ambiguities(ImageInTerminal, Base, Core)
 using ReferenceTests
-ambs = detect_ambiguities(ReferenceTests, ImageInTerminal, Base, Core)
+ambs = detect_ambiguities(ReferenceTests, ImageInTerminal, Base, Core)|
```

Comment:
whats the `|` for ?

Reply:
Arg,
I am clumsy, and mash the | key because it is near the enter key